src/main/java/lotterymachine/model/Count.java

```diff
@@ -0,0 +1,22 @@
+package lotterymachine.model;
+
+public class Count {
+    private int number;
+
+    public Count(int number) {
+        this.number = number;
+    }
+
+    public static Count from(int number) {
+        return new Count(number);
+    }
```

Comment:
```suggestion
    private Count(int number) {
        this.number = number;
    }

    public static Count from(int number) {
        return new Count(number);
    }
```
정적팩토리로 생성을 유도하려면, 생성자를 막는게 부수효과를 줄일 수 있을 것 같아요.

Reply:
좋은 피드백 감사합니다!
`Count` 클래스 뿐만 아니라 정적 팩토리 메서드를 선언하고 사용하는 `Money`와 `LotteryResultDto`에도 적용하였습니다.